modules/tinymce/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Improved
 - Direct invalid child text nodes of list elements will be wrapped in list item elements. #TINY-4818
 
+### Fixed
+- Ranged deletion of formatted text using selection or keyboard shortcut would sometimes cause Blink and Webkit browsers to insert interpreted tags upon typing, which may affect document semantics. #TINY-9302
```

Comment:
We might want to choose something a bit more straightforward than "affect document semantics"

Reply:
Do you mean I should give the example of `<u>` replacing `<span style="text-decoration: underline;>`?

Reply:
I'm not sure exactly. It's just that "affect document semantics" just sounds very formal, and isn't really all that clear. It's hard because it doesn't create deprecated tags ... maybe inconsistent tags?

Reply:
Wording clarified:
~affect document semantics~ -> result in inconsistent tags